pkgs/development/tools/open-policy-agent/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "open-policy-agent";
+  version = "v0.9.2";
```

Comment:
```suggestion
  version = "0.9.2";
```
Version [must start with a digit](https://nixos.org/nixpkgs/manual/#sec-package-naming).

Reply:
Oops, I know it moreover :/